From 5f8550babba505c4a3333f5a2e6e3fb8047db061 Mon Sep 17 00:00:00 2001
From: PavelBARABANOV <pavelbarabanov94@gmail.com>
Date: Sun, 18 Oct 2026 22:21:52 +0300
Subject: [PATCH 25/56] no need

(cherry picked from commit 99ad87d88591de8dec6098cf2de59d3dacc4ea2c)
---
 src/core/file_sys/registered_cache.cpp | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/src/core/file_sys/registered_cache.cpp b/src/core/file_sys/registered_cache.cpp
index 99a6edcc5..defc3d7d9 100644
--- a/src/core/file_sys/registered_cache.cpp
+++ b/src/core/file_sys/registered_cache.cpp
@@ -1460,12 +1460,6 @@ void ExternalContentProvider::ProcessNSP(const VirtualFile& file) {
         return;
     }
 
-    // Reject solid NCZ (.nsz without -B): Eden only supports block-compressed NCZ.
-    if (!ContainerIsFullyBlockCompressed(nsp)) {
-        LOG_DEBUG(Service_FS, "Skipping solid-compressed container: {}", file->GetName());
-        return;
-    }
-
     LOG_DEBUG(Service_FS, "Processing NSP file: {}", file->GetName());
     AddExternalEntriesFromContainer(nsp, entries, versions, multi_version_entries);
 }
@@ -1476,11 +1470,6 @@ void ExternalContentProvider::ProcessXCI(const VirtualFile& file) {
         return;
     }
 
-    if (!ContainerIsFullyBlockCompressed(nsp)) {
-        LOG_DEBUG(Service_FS, "Skipping solid-compressed container: {}", file->GetName());
-        return;
-    }
-
     AddExternalEntriesFromContainer(nsp, entries, versions, multi_version_entries);
 }
 
-- 
2.43.0

